Automattic/jetpack
# Extensions: Simplify directories

Fixes a small thread of dignity

#### Changes proposed in this Pull Request:

* Removes `extensions/setup` and puts its content in either the root of the extensions folder, or in `extensions/shared`
* Removes `extensions/utils` and puts its content in `extensions/shared`

#### Testing instructions:

- run `yarn build` and get no errors
- verify that extensions in the post editor look ok

#### Proposed changelog entry for your changes:

* None
